lib/vtls/schannel.c

```diff
@@ -284,8 +290,6 @@ get_alg_id_by_name(char *name)
   defined(__MINGW64_VERSION_MAJOR) ||                                   \
   (__W32API_MAJOR_VERSION > 5)     ||                                   \
   ((__W32API_MAJOR_VERSION == 5) && (__W32API_MINOR_VERSION > 0))
-  /* CALG_TLS1PRF has a syntax error in MinGW's w32api up to version 5.0,
```

Comment:
I don't understand. What was the build error? The preprocessor condition here was meant to avoid build errors. Or was the problem the lack of this `CIPHEROPTION`? Then this preprocessor condition should be deleted.

Reply:
It's the same build error as described in the [bug](https://osdn.net/projects/mingw/ticket/38391). The current fix in schannel does not work here because __W32API_MAJOR_VERSION is undefined. It is defined in w32api.h but that is not included by any other mingw include. I could get it working with:

#ifdef __MINGW32__
#include <w32api.h>
#endif

However rather than do that to fix the version check the latest iteration of my patch removes it and compensates for the typo instead.